### ISSUE TITLE
[Bug] Logging in displays: auth_cookie_insecure

### ISSUE THREAD

**Issue by kroese** [LemmyNet/lemmy-ui#2193]
### Requirements

- [X] This is a bug report, and if not, please post to https://lemmy.ml/c/lemmy_support instead.
- [X] Please [check](https://github.com/LemmyNet/lemmy-ui/issues) to see if this issue already exists.
- [X] It's a single bug. Do not report multiple bugs in one issue.
- [X] It's a frontend issue, not a backend issue; Otherwise please create an issue on the [backend repo](https://github.com/LemmyNet/lemmy) instead.

### Summary

In Lemmy 0.19.0-beta.8 when you sign in for the first time, it works fine. But as soon as you logout and try to signin for a second time, it displays the message:

`auth_cookie_insecure`

This error does not go away until you clear your cache/cookies. 

As a workaround I always need to use an incognito tab now.

### Steps to Reproduce

1. Sign in
2. Sign out
3. Try to sign in
4. auth_cookie_insecure error

### Technical Details

Samsung Internet browser on Android. But also with Edge on Win11.

### Lemmy Instance Version

0.19.0-beta.8

**Comment by TheSaneWriter** [comment]
Indeed, I'm able to reproduce this issue on lemmy.thesanewriter.com on 0.19.0-rc.3

**Comment by fantismo** [comment]
Happened to me on 0.19.0-rc.4.
